src/components/FooterPage/Footer.scss

```diff
@@ -0,0 +1,41 @@
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0px;
+    padding: 0px;
+    overflow-x: hidden;
+}
+
+
+.footer_bg {
+
+
+    background-color: #59BDEE !important;
+}
+
+
+.copyright {
```

Comment:
as for this section th !important overrride is not needed anywhere in this file. This could lead to overrides in places you might not of thought of. [See attached error] 
![error1](https://user-images.githubusercontent.com/18339677/109569535-9bb68a00-7ab6-11eb-990f-b729cdc10564.jpg) The '!important override cause the a tag for the whole page to turn blue. and we definitely do not want that to happen. Please remove all !important tags in this code

Reply:
Thanks! I removed it, hopefully this resolve the error